src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/AvailableDelegationsTests.ps1

```diff
@@ -0,0 +1,33 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Tests checking API to list available delegations.
+#>
+function Test-GetAvailableDelegationsList
+{
+    $location = Get-ProviderLocation ResourceManagement
+    $location = $location -replace " ",""
```

Comment:
Your service should be able to handle both Display name and shortened name of an Azure region.  Is there a reason it does not? Users will provide both.

Reply:
it handles both (I verified this). This test code was common elsewhere, it's a product of c&p.